tests/test_run_integration_tests.cairo

```diff
@@ -133,12 +133,6 @@ func test_non_eth_round_trip{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, ran
     return ();
 }
 
-@external
-func test_deposit_liquidity{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(){
```

Comment:
why is this whole function just deleted

Reply:
because the test doesn't actually do anything (see the deleted file)

Reply:
i don't see a connection to test_deposit func, that is a legitimate test